src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -765,6 +823,33 @@ void P256Keypair::Clear()
     }
 }
 
+CHIP_ERROR P256Keypair::Copy(const P256Keypair & other)
+{
+    CHIP_ERROR error    = CHIP_NO_ERROR;
```

Comment:
This class can be used for both ECDH and ECDSA, so it must be documented that this method assumes that it should only be called for ECDSA keys.